loadtest/report/compare.go

```diff
@@ -50,41 +55,7 @@ func Compare(target *os.File, genGraph bool, reports ...Report) error {
 	// TODO: generate a single image combining all the graphs.
 	// Printing the graphs.
 	if genGraph {
-		// gPlots is a slice of structs to aggregate graphs of a single type
-		// from multiple reports.
-		var gPlots []struct {
-			name   string // the name of the metric
-			graphs []labelValues
-		}
-		// A single report has multiple graphs.
-		// What we are doing here is changing the aggregation such that
-		// a single gPlot contains graphs of a single type from multiple reports.
-		for i, r := range reports[1:] {
-			for _, g := range r.Graphs {
-				if i == 0 {
-					gPlots = append(gPlots, struct {
-						name   string
-						graphs []labelValues
-					}{
-						// Only set the name one time
-						name: g.Name,
-						graphs: []labelValues{
-							{
-								label:  r.Label,
-								values: g.Values,
-							},
-						},
-					})
-				} else {
-					// Graph of the same type from another report, append it.
-					gPlots[i].graphs = append(gPlots[i].graphs, labelValues{
-						label:  r.Label,
-						values: g.Values,
-					})
-				}
-			}
-		}
-
+		gPlots := getPlots(reports...)
```

Comment:
Does it make sense to pass reports without the base here? So that `getPlots` can work on a slice of reports without worrying about this implementation detail.